server.js

```diff
@@ -482,8 +482,11 @@ function bindClientEventBridge() {
     });
 
     mavlinkParser.on('STATUSTEXT', function(message) {
-        io.emit('STATUSTEXT', message.text);
-        logger.info('status text from APM: ' + util.inspect(message.text));
+        // v8 doesnt have the regex global flags anymore, so use underscore to remove them
+        var statusText = _.filter(message.text, function(char){ return char !== '\u0000'; }).join('').toString();
```

Comment:
It may feel premature, but I'm a little worried about the performance of this -- we know there's not too many STATUSTEXT messages, but would [something like this](https://stackoverflow.com/questions/10297683/how-to-remove-u0000-from-string) work instead?  Maybe just file a ticket and forget it for now?